client/src/components/mapPopup.tsx

```diff
@@ -0,0 +1,130 @@
+import React, {useRef, useEffect, useState} from 'react';
+import * as styles from './mapPopup.module.scss';
+import Overlay from 'ol/Overlay';
+import {Table} from '@trussworks/react-uswds';
+import {Coordinate} from 'ol/Coordinate';
+import Map from 'ol/Map';
+import {FeatureLike} from 'ol/Feature';
+
+interface IMapPopupProps {
+    map: Map,
+    selectedFeature: FeatureLike;
+    position: Coordinate;
+}
+
+const MapPopup = ({map, selectedFeature, position}: IMapPopupProps) => {
+  const popupContainerElement = useRef<HTMLDivElement>(null);
+  const popupCloserElement = useRef<HTMLAnchorElement>(null);
+  const popupContentElement = useRef<HTMLDivElement>(null);
+  const [currentOverlay, setCurrentOverlay] = useState<Overlay>();
+
+  const readablePercent = (percent: number) => {
+    return `${(percent * 100).toFixed(2)}%`;
+  };
+
+  useEffect(() => {
+    if (!popupCloserElement && !popupContainerElement) return;
+    popupCloserElement.current!.onclick = function() {
+      overlay.setPosition(undefined);
+      popupCloserElement.current!.blur();
+      return false;
+    };
+
+    const overlay = new Overlay({
+      element: popupContainerElement.current!,
+      autoPan: true,
+      autoPanAnimation: {
+        duration: 250,
+      },
+    });
+    setCurrentOverlay(overlay);
+    map.addOverlay(overlay);
+  }, []);
+
+  useEffect( () => {
+    if (position && currentOverlay) { // may be empty on first render
+      currentOverlay.setPosition(position);
+    }
+  }, [position]);
+
+  const getCategorization = (percentile: number) => {
+    let categorization = '';
```

Comment:
Doesn't need to be initialized since it's always set below. (I don't care, but my linter noticed it.)